website_compassion/template/my_account_my_children_donate.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+     <data>
+        <template id="my_account_my_children_donate" name="Donate to sponsored children">
+            <t t-call="website_compassion.my_account_layout">
+                <t t-set="javascript_hook"> my_children_donate_select_child</t>
+                <t t-set="javascript_param" t-value="''"/>
+                <t t-call="website_compassion.children_horizontal_cards"/>
+                <center>
+                    <h3>Make a gift to <t t-esc="child_id.preferred_name"/></h3>
```

Comment:
```suggestion
                    <h3>donation_form.form_title()</h3>
```
To keep all the code in one place you could define the form title directly in python (there is also a form_description if needed)